freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>freeorion.desktop</id>
+  <name>FreeOrion</name>
+  <summary>Turn-based space empire and galactic conquest (4X) computer game</summary>
+  <summary xml:lang="de">Rundenbasierendes Weltraum-Strategiespiel</summary>
+  <developer_name>FreeOrion Project</developer_name>
```

Comment:
Do we have a formal developer name? @freeorion/developers

Reply:
FreeOrion Project is fine, I think...

Reply:
Yep, FreeOrion Project should be ok. That's kind of our "formal developer name"...